### PR TITLE
Markup 1

### PR DESCRIPTION
**Pull request by Bachyla**

### CODE REVIEW

File: Markup-1/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<h1 align="center">Lambda</h1><br>
```
Comment (sleepwalky) [inline Markup-1/README.md]: For me it's quite strange to see html markup in markdown file. Is it common approach?

File: Markup-1/css/styles.css [hunk 2]
```diff
@@ -0,0 +1,1376 @@
...
+}
+
+img {
+	border-style: none;
+}
+
+body {
+	font-family: 'Lato', sans-serif;
+	font-size: 16px;
+	font-size: 1rem;
```
Comment (sleepwalky) [inline Markup-1/css/styles.css]: Do you really need to have font-size rule both for px and rem units?

File: Markup-1/css/styles.css [hunk 3]
```diff
@@ -0,0 +1,1376 @@
...
+		width: 100px;
+		font-size: 20px;
+	}
+}
+
+
+/* ---------------------- SVG ICONS ---------------------- */
+
+.social-facebook {
+	background-image: url(data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0idXRmLTgiPz4KPCEtLSBHZW5lcmF0ZWQgYnkgSWNvTW9vbi5pbyAtLT4KPCFET0NUWVBFIHN2ZyBQVUJMSUMgIi0vL1czQy8vRFREIFNWRyAxLjEvL0VOIiAiaHR0cDovL3d3dy53My5vcmcvR3JhcGhpY3MvU1ZHLzEuMS9EVEQvc3ZnMTEuZHRkIj4KPHN2ZyB2ZXJzaW9uPSIxLjEiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgeG1sbnM6eGxpbms9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkveGxpbmsiIHdpZHRoPSI5IiBoZWlnaHQ9IjE2IiB2aWV3Qm94PSIwIDAgOSAxNiI+CjxwYXRoIGZpbGw9IiNGRkZGRkYiIGQ9Ik04LjU2MyAwLjEwN3YyLjM1N2gtMS40MDJxLTAuNzY4IDAtMS4wMzYgMC4zMjF0LTAuMjY4IDAuOTY0djEuNjg4aDIuNjE2bC0wLjM0OCAyLjY0M2gtMi4yNjh2Ni43NzdoLTIuNzMydi02Ljc3N2gtMi4yNzd2LTIuNjQzaDIuMjc3di0xLjk0NnEwLTEuNjYxIDAuOTI5LTIuNTc2dDIuNDczLTAuOTE1cTEuMzEzIDAgMi4wMzYgMC4xMDd6Ij48L3BhdGg+Cjwvc3ZnPgo=);
```
Comment (sleepwalky) [inline Markup-1/css/styles.css]: https://docs.google.com/presentation/d/1z49qp03iXAJIkbXaMtCmWW_Pnnq-MzXGW139Xw8-paM/edit#slide=id.g174590d5d_0194

**Comment by sleepwalky** [comment]
You can also take a look into https://www.youtube.com/watch?v=-_9rsXq6OvE&list=PLe--kalBDwjiXq7_8kUgcMmZJpix9ITkK&index=11

File: Markup-1/css/styles.css (hunk 2 above)
Comment (bachilo) [reply Markup-1/css/styles.css]: Yeah, I think you're right. I don't need it.

File: Markup-1/css/styles.css (hunk 3 above)
Comment (bachilo) [reply Markup-1/css/styles.css]: Thank you!
I will not make mistakes in the future.

File: Markup-1/README.md (hunk 1 above)
Comment (bachilo) [reply Markup-1/README.md]: So, I don't even know what to say to that. 
I will consider this issue!

**Pull request merged**